src/CentrifugeGasService.sol

```diff
@@ -0,0 +1,57 @@
+// SPDX-License-Identifier: AGPL-3.0-only
+pragma solidity 0.8.21;
+
+import {Auth} from "src/Auth.sol";
+import {IAggregatorV3} from "src/interfaces/IAggregatorV3.sol";
+
+contract CentrifugeGasService is IAggregatorV3, Auth {
+    uint8 public constant decimals = 18;
+    string public constant description = "CFG/ETH price feed";
+    uint256 public constant version = 1;
+
+    uint80 roundId;
+    uint256 updatedTime;
+    uint256 centrifugePrice; // decide  in what denomination
+    uint256 centrifugeCost;
+
+    event File(bytes32 what, uint256 value);
+
+    constructor(uint256 centrifugeCost_, uint256 centrifugePrice_) {
+        centrifugeCost = centrifugeCost_;
+        centrifugePrice = centrifugePrice_;
+        updatedTime = block.timestamp;
+        roundId = 1;
+    }
+
+    function file(bytes32 what, uint256 value) external auth {
+        if (what == "centrifugeCost") {
+            centrifugeCost = value;
+        }
+        if (what == "centrifugePrice") {
```

Comment:
nit:
```suggestion
        if (what == "cost") {
            cost = value;
        }
        if (what == "price") {
```

Reply:
Agree. In general we have to strive for simplicity in naming. Since there is no specific need to use `centrifuge` to distinguish the variable ( like two price variables or cost variables ) we can just use simple words.